compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/generators/CallGenerator.kt

```diff
@@ -328,7 +331,18 @@ internal class CallGenerator(
             is IrFieldExpressionBase -> {
```

Comment:
Hm, github doesn't allow me to leave comments outside of changes. :\

Anyhow, I saw that applying the same idea to `IrCallImpl` case above, in particular, `dispatchReceiver` for call expression without selector could also unmute 17 cases, but broke many others. :p It may require more careful accessibility check I guess. I'll dig into that as well, if this approach looks good.